src/goto-programs/link_to_library.h

```diff
@@ -18,19 +18,11 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/deprecate.h>
 #include <util/irep.h>
 
-class goto_functionst;
 class goto_modelt;
 class message_handlert;
 class symbol_tablet;
 
 DEPRECATED(SINCE(2019, 2, 28, "Use link_to_library(goto_model, ...) instead"))
-void link_to_library(
```

Comment:
Shouldn't that `DEPRECATED` line go away?

Reply:
Thank you for catching this!